editor/modes/visual-editor/block.js

```diff
@@ -129,7 +128,8 @@ class VisualEditorBlock extends wp.element.Component {
 					}
 				},
 				...blocksWithTheSameType.slice( 1 )
-			]
+			],
+			{ offset: -1 }
```

Comment:
Yes, for this. I'd use something like this instead https://github.com/tappleby/redux-batched-subscribe 

If it's not creating any bug to call two actions, I'd prefer if we solve this separately.